.vuepress/add_placeholders.rb

```diff
@@ -1,30 +1,32 @@
-# This function will add placeholders for pages that are out of scope of the docs preview
```

Comment:
Not sure what this big diff is about?

Reply:
I have added the new placeholders here but then introduced them in another pr (because of the failing build) and rebased this branch.

Still don't know why it shows a complete diff, but the contents should be the same.